Rails-Girls-Summer-Of-Code-2017.md

```diff
@@ -0,0 +1,56 @@
+----------
+
+# Team 276linesofCode - Rails Girls Summer of Code 2017
+
+Hey all,
+
+It has been a long long ride since February and it's finally July!
+This is when we start our Rails Girls Summer of Code (RGSoC) 2017 program, by working and contributing for Tessel. Having seen Tessel as a project in the RGSoC project list of 2017, we were eager to take it up at once since, we felt that through this project we would be able to discover both the horizons - Hardware and Software.
+
+
+### Who are we?
+
+<p align="center">
+<img src="https://github.com/276linesofCode/blog-posts/blob/master/IMG_3774.JPG" width="200">
+</p>
+
+Team 276linesofCode is an amalgamation of all things tech and electronic. The Team comprises Shravika Mittal and Brihi Joshi, a duo from New Delhi, India, in their sophomore year of College at IIIT Delhi. We are a dynamic duo that believes in learning and strengthening concepts while on the way to developing something.
+
+Shravika is an Undergraduate majoring in Computer Science and Engineering. It's been hardly a year since she's got her hands dirty with programming, and it's already a vital part of her daily schedule. She brings out dedication and hard work in the team, excelling in whatever she does.
+
+Brihi is majoring in Electronics and Communication Engineering and loves to mix and fiddle with creativity in code. She brings in the caffeine-induced, late night coding sessions along with fresh experiments in the team.
+
+
+### What is [RGSoC](https://railsgirlssummerofcode.org/)?
+
+Rails Girls Summer of Code is a three month long award-winning global fellowship program aimed at bringing more diversity into Open Source.
+
+It is about helping __introduce newcomers__ to the world of programming further expand their knowledge and skills, by contributing to a worthwhile Open Source project. The focus is not on producing highly sophisticated code, but rather participants learning transferable skills from their project work. Apart from inducing coding skills in real world projects, the RGSoC inculcates community building, by introducing its scholars to the mentors who are in control of the actual projects.
```

Comment:
Have another look at that first sentence, I think sentence structure got lost in editing